modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/sql/DistributionZoneSqlDdlParserTest.java

```diff
@@ -291,13 +291,15 @@ public void alterZoneIfExistsSet() {
      */
     @Test
     public void alterZoneSetOptions() {
-        IgniteSqlAlterZoneSet alterZoneSet = parseAlterZoneSet(
-                "alter zone a.test_zone set "
-                        + "replicas=2, "
-                        + "data_nodes_filter='(\"US\" || \"EU\") && \"SSD\"', "
-                        + "data_nodes_auto_adjust=1, "
-                        + "data_nodes_auto_adjust_scale_up=2, "
-                        + "data_nodes_auto_adjust_scale_down=3"
+        IgniteSqlAlterZone alterZoneSet = parseAlterZone(
+                "alter zone a.test_zone set ("
+                        + "REPLICAS 2, "
+                        + "PARTITIONS 3, "
+                        + "DISTRIBUTION ALGORITHM 'test_Distribution', "
```

Comment:
change of partitions as well as change of distribution algorithm is not allowed

Reply:
got it, changed